internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -355,64 +324,66 @@ func setIRQLoadBalancing(c *oci.Container, enable bool, irqSmpAffinityFile, irqB
 	return nil
 }
 
-func setCPUQuota(cpuMountPoint, parentDir string, c *oci.Container, enable bool) error {
-	var rpath string
-	var err error
-	var cfsQuotaPath string
-	var parentCfsQuotaPath string
-	var cgroupManager cgmgr.CgroupManager
+func setCPUQuota(parentDir string, c *oci.Container) error {
+	var (
+		cgroupManager cgmgr.CgroupManager
+		err           error
+	)
 
 	if strings.HasSuffix(parentDir, ".slice") {
-		// systemd fs
 		if cgroupManager, err = cgmgr.SetCgroupManager("systemd"); err != nil {
```

Comment:
Can we use a constant to set it?

Reply:
we can but I'll not do it here and start the PR in the main branch. this is a cherry-pick